pyproject.toml

```diff
@@ -116,6 +116,7 @@ optional = [
 test = [
     'asv',
     'matplotlib>=3.5',
+    'numpydoc>=1.5',
```

Comment:
This also needs to go into `requirements/test.txt`. It think that's why the tests don't find numpydoc. The pre-commit hook should also complain about this.

Reply:
How antiquated... 😂 Ok, I'll push up an update in the morning...